src/libLookup/Lookup.cpp

```diff
@@ -697,7 +699,11 @@ void Lookup::SendMessageToLookupNodes(const zbytes& message) const {
     }
   }
 
-  P2PComm::GetInstance().SendBroadcastMessage(allLookupNodes, message);
+  // TODO use distributed traces from here?
```

Comment:
this todo is copied across the PR many times

Reply:
Talk pvt.